lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -36,7 +36,7 @@
   border: $nx-border;
```

Comment:
I've just noticed that something about this placeholder class isn't working correctly.  I'm seeing 9 identical blocks in the CSS output that have a selector of `.nx-code` and these styles.

Reply:
> I've just noticed that something about this placeholder class isn't working correctly. I'm seeing 9 identical blocks in the CSS output that have a selector of `.nx-code` and these styles.

I moved all of the helper CSS files out of the individual style components' SCSS files and into `_globals.scss` which seems to have fixed the issue: https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/38097ad8947e4f63d40345da034c441b669333b5

Reply:
> I moved all of the helper CSS files out of the individual style components' SCSS files and into _globals.scss which seems to have fixed the issue: 38097ad

Just within the base styles, you mean, right?